protocol/package.json

```diff
@@ -2,11 +2,13 @@
   "name": "truco-protocol",
   "devDependencies": {
     "@nomicfoundation/hardhat-toolbox": "^2.0.0",
+    "@nomiclabs/hardhat-ethers": "^2.2.1",
     "@nomiclabs/hardhat-etherscan": "^3.1.1",
     "@openzeppelin/hardhat-upgrades": "^1.21.0",
     "@typechain/ethers-v5": "^10.1.0",
     "@typechain/hardhat": "^6.1.3",
     "dotenv": "^16.0.3",
+    "ethers": "^5.7.2",
```

Comment:
`ethers` is a direct dependency of `hardhat`, which is already declared as a dependency